sdk/identity/Azure.Identity/src/AzureApplicationCredentialOptions.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+
+namespace Azure.Identity
+{
+    /// <summary>
+    /// Options to configure the <see cref="AzureApplicationCredential"/> authentication flow and requests made to Azure Identity services.
+    /// </summary>
+    public class AzureApplicationCredentialOptions : TokenCredentialOptions
```

Comment:
If the goal of the options bag is to allow users to setup the credential via code should they be able to set the client id, tenant id, and secret or certificate? I'm not sure what the best way to set these would be because they are somewhat mutually exclusive. Perhaps different constructor overloads?

Reply:
Let's leave out this extra code configuration for this release and add an issue to investigate adding it before the final beta.

Reply:
#23242